versions/development/SPEC.md

```diff
@@ -487,7 +488,7 @@ Below are the valid results for operators on types.  Any combination not in the
 |`Boolean`|`<=`|`Boolean`|`Boolean`||
 |`Boolean`|`||`|`Boolean`|`Boolean`||
 |`Boolean`|`&&`|`Boolean`|`Boolean`||
-|`File`|`+`|`File`|`File`|Append file paths|
+|`File`|`+`|`File`|`File`|Concatenate file paths|
```

Comment:
This has always made no sense to me. IMO `File + File` should be removed

Reply:
I think you are right. It would simplify this table, and clarify much of the spec, if the way that string and File were related was explicit. For example: ```Files will be coerced into Strings, and Strings into files, based on the destination type of the expression```. One issue is that some of the library functions have different meaning for string and file -> read_json from a string, or from the file named by this 'String(File)'

Reply:
Oh and I meant to say, step 2: ```String + String = String``` would then be the only defined + operator.

Reply:
+1 to removing File + File. It really does not make much sense to me. If `FIle` was treated as any `URI` or `PATH` then it might make snese. But in this case any `File` is an actual location in the cloud or on the local instance. Concatenating them is silly..